src/material/core/_core.scss

```diff
@@ -16,20 +22,31 @@
   @include text-field.text-field-autosize();
   @include text-field.text-field-autofill();
 
-  @include focus-indicators.private-strong-focus-indicators-positioning();
-  @include _mdc-core();
-}
+  .mat-focus-indicator {
+    @include _structural_styling('mat');
+  }
 
-// Mixin that renders all of the core MDC styles. Private mixin included with `mat-core`.
-@mixin _mdc-core() {
-  @include _mdc-strong-focus-indicators-positioning();
+  .mat-mdc-focus-indicator {
+    @include _structural_styling('mat-mdc');
+  }
 }
 
-// Mixin that ensures focus indicator host elements are positioned so that the focus indicator
-// pseudo element within is positioned relative to the host. Private mixin included within
-// `_mat-mdc-core`.
-@mixin _mdc-strong-focus-indicators-positioning() {
-  .mat-mdc-focus-indicator {
-    position: relative;
+@mixin _structural_styling($library) {
```

Comment:
We generally keep the `core.scss` pretty minimal. It may be more appropriate to create another file in `src/material/core/style` specifically for the focus indicators.

Reply:
Sounds good! Will create and move the `structural_styling` mixin to `src/material/core/style/_focus-indicators.scss`